test/table_test.rb

```diff
@@ -148,8 +148,8 @@ def test_change_value_then_save_again_should_noop
   def test_updates_fields_to_newest_values_after_update
     record = first_record
 
-    record["Name"] = "new_name"
-    stub_patch_request(record, ["Name"], return_body: record.fields.merge("Notes" => "new animal"))
+    record[:name] = "new_name"
```

Comment:
Shouldn't this be a string @Meekohi as we're deprecating symbol access?

Reply:
Yup -- still getting the hang of all these review features I must have mixed something up with one of those commits.
```suggestion
    record["Name"] = "new_name"
```